README.md

```diff
@@ -41,121 +46,187 @@ You can find a version of this docummentation, properly rendered, at https://doc
+In this section we will cover a mix of some commands and interfaces available.
+
+**Note:**
+The first time you run some of these commands, the source{d} Engine will download and install the Docker containers as needed. Be aware that this might take a bit of time if it is the first time you use them.
+
+#### Querying code
+
+##### Query web interface
 
 To launch the [web client for the SQL interface](https://github.com/src-d/gitbase-web), run the following command and start executing queries:
 
 ```bash
+# Launch the query web client
 srcd web sql
 ```
 
-The first time you run some of these commands, the source{d} Engine will download and install the Docker containers that are needed. Be aware that this might take a bit of time, it is only on your first use. 
+This should open the [web interface](https://github.com/src-d/gitbase-web) in your browser.
+You can also access it directly at [http://localhost:8080](http://localhost:8080).
 
-If you prefer to stay with the command line, you can execute:
+##### Query command-line interface (CLI)
+
+If you prefer to work within your terminal via command line, you can open a SQL REPL
+that allows you to execute queries against your repositories by executing:
 
 ```bash
+# Launch the query CLI REPL
 srcd sql
 ```
 
-This will open a SQL REPL that allows you to execute queries against your repositories.
-
 If you want to run a query directly, you can also execute it as such:
 
 ```bash
+# Run query via CLI
 srcd sql "SHOW tables;"
 ```
 
-You might have noticed that some queries below use the UAST function. This is to transform code to a [Universal Abstract Syntax Tree](#babelfish-uast). If you want a playground to see examples of the UAST, or run your own, you can launch the parse web client.
+**Note:**
+Queries using the [UAST](#babelfish-uast) function are meant for the `web sql` interface.
+The column for UAST will be seen as binary data in the CLI.
+
+#### Parsing code
+
+Sometimes you may want to parse files directly as [UASTs](#babelfish-uast).
 
-**Note:** queries using the UAST function are meant for the `web sql` interface. The column for UAST will be seen as binary data in the CLI.
+To see which languages are available, check the table of [supported languages](#babelfish-uast).
 
-To see which languages are available, check the table of [supported languages](https://docs.sourced.tech/babelfish/languages).
+##### Parse web client
 
-The first time you launch the web client, it will download and install the recommended drivers (those that are beta or higher). 
+If you want a playground to see examples of the UAST, or run your own, you can launch the [parse web client](https://github.com/bblfsh/web).
+
+The first time you launch the web client, it will download and install the recommended language drivers (those that are beta or higher):
```

Comment:
This is not true anymore, we changed the code but missed this in the docs.
Now we install the bblfsd version that comes with all the recommended drivers bundled. I would just remove this sentence, or change it to "The first time you launch the web client, it will download and install the needed components."